pika/adapters/select_connection.py

```diff
@@ -439,9 +436,9 @@ def add_callback_threadsafe(self, callback):
 
```

Comment:
Also need to add the efficiency blurb in `IOLoop.add_callback_threadsafe()` docstring (just like the one you added in `BlockingConnection.add_callback_threadsafe()`). Except that in IOLoop, it's `call_later()` instead of `add_timeout()` and `delay` instead of `deadline`.

Reply:
Before `1.0.0` is released should these three methods all have the same name and signature?